### DISCUSSION TITLE
mitmproxy does not show https traffic.

### DISCUSSION THREAD

**Discussion by a-poliusov** [mitmproxy/mitmproxy#6663] [Q&A]
Hi. So I installed mitmproxy and certificate. 
I can open mitm.it page and it does show certificates to pick.
But when I open google, it does not anything.
https://discourse.mitmproxy.org/t/os-x-terminal-traffic-doesnt-automatically-appear-in-mitmproxy/103/3
there is a thread I found and it seems like I have the same problem, but the fix they propose does not work for me.

So if I curl with 

I get 

`curl --proxy localhost:8080 'https://mitmproxy.org'`

```
  GET https://alive.github.com/_sockets/u/157006913/ws?session=eyJ2IjoiVjM…
       ← 101 [no content] 169ms
>> GET https://mitmproxy.org/ HTTP/2.0
       ← 200 text/html 23.4k 78ms
```
also you can see some github traffic, but it ends with something weird. 

What am I doing wrong? Please help me, I'm so tired figuring it out.

**Comment by emanuele-em** [comment]
Hi!
probably stupid question: mitm.it is visible also if you just proxy the `http` traffic but you also have to proxy `https` traffic by explicitly selecting it, have you done that?
On macOS:
<img width="550" alt="image" src="https://github.com/mitmproxy/mitmproxy/assets/100081325/cb5902c0-30b6-49c1-bb33-080458223162">
